Detectors/EMCAL/base/src/Geometry.cxx

```diff
@@ -897,12 +882,11 @@ void Geometry::CreateListOfTrd1Modules()
   }
 }
 
-ShishKebabTrd1Module* Geometry::GetShishKebabModule(Int_t neta) const
+const ShishKebabTrd1Module &Geometry::GetShishKebabModule(Int_t neta) const
```

Comment:
Currently rules allow to place the operator either on the left or on the right side. Will stay with the current definition until the coding conventions become more specific with respect to this point.